modules/node/src/transfer/transfer.service.ts

```diff
@@ -228,8 +232,9 @@ export class TransferService {
               assetId,
             );
             if (fb[freeBalanceAddr].lt(amountBN)) {
-              // wait for resolve
-              return;
+              reject(
+                `Free balance associated with ${freeBalanceAddr} is less than transfer amount: ${amountBN}`,
+              );
```

Comment:
fine to reject here (imo) because we don't ever re-request a deposit into the channel so the payment would fail anyways.

we may want to handle that edge case better